src/NuGet/Microsoft.Net.Compilers.Toolset/build/Microsoft.Net.Compilers.Toolset.props

```diff
@@ -0,0 +1,17 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project DefaultTargets="Build">
+
+  <PropertyGroup>
+    <_TargetDirectoryName Condition="'$(MSBuildRuntimeType)' == 'Core'">netcoreapp2.1</_TargetDirectoryName>   
+    <_TargetDirectoryName Condition="'$(MSBuildRuntimeType)' != 'Core'">net472</_TargetDirectoryName>   
+    <_TasksDirectory>$(MSBuildThisFileDirectory)..\tasks\$(_TargetDirectoryName)\</_TasksDirectory>
```

Comment:
>_TasksDirectory [](start = 5, length = 15)

I suggest _RoslynTasksDirectoy, _RoslynTargetDirectoryName here. These will be visible to everyone else and the names are super generic.

Reply:
The intent is to make them visible. See Bootstrap.targets where we use the name. There are already other Roslyn prefixed variables in the build. That's why I chose this one here.

Reply:
I'm referring to the _ ones that are not Roslyn prefixed. I would put _Roslyn (or anything to differentiate).

Reply:
Oh I thought `_` was the way to differentiate. I though it was MSBuild convention for `private`.

Reply:
It is, but it's just a convention, so I would be slightly more defensive. It also makes it easier to find things in the binlog if you don't use super generic names in product targets for these sorts of things.

Reply:
Gotcha. Fixed.